src/cascade/executor/epiviz_runner.py

```diff
@@ -136,16 +178,30 @@ def main():
 
 
 def entry():
+    parser = DMArgumentParser("Run DismodAT from Epiviz")
+    parser.add_argument("db_file_path")
+    parser.add_argument("--settings_file")
+    parser.add_argument("--no-upload", action="store_true")
+    parser.add_argument("--pdb", action="store_true")
+    args, _ = parser.parse_known_args()
+
     try:
-        main()
+        main(args)
     except SettingsError as e:
-        CODELOG.error(str(e))
-        CODELOG.error(f"Form data: {pformat(e.form_data)}")
-        CODELOG.error(f"Form validation errors: {pformat(e.form_errors)}")
+        MATHLOG.error(str(e))
+        MATHLOG.error(f"Form data: {pformat(e.form_data)}")
+        MATHLOG.error(f"Form validation errors: {pformat(e.form_errors)}")
         exit(1)
     except Exception:
-        CODELOG.exception(f"Uncaught exception in {os.path.basename(__file__)}")
-        raise
+        if args.pdb:
+            import pdb
+            import traceback
+
```

Comment:
In the right direction here.

Reply:
This is a bit of tooling I love. I figured it out for vivarium and stole it from there for us. It should probably go in your argparse code but that didn't have a clear place to put it and I didn't have time to explore.